lib/actionpack/all/actionpack.rbi

```diff
@@ -97,8 +97,8 @@ class ActionController::Parameters
   sig { params(key: T.any(String, Symbol)).returns(T::Boolean) }
   def key?(key); end
 
-  sig { params(key: T.any(String, Symbol)).returns(T.untyped) }
-  def keys(key); end
+  sig { returns(T.untyped) }
+  def keys; end
```

Comment:
weren't arguments for this signature autogenerated?
Said otherwise, how sure are you that the sig is correct?

Reply:
> weren't arguments for this signature autogenerated?

I believe they were added by hand here: https://github.com/sorbet/sorbet-typed/pull/111

> are you sure that the changed argument list is correct?

If you provide more than 0 args to `keys`, it's an ArgumentError:

```ruby
> ActionController::Parameters.new(foo: 'bar').keys(:foo)
ArgumentError: wrong number of arguments (given 1, expected 0)
```

Reply:
Thank you!